setup.py

```diff
@@ -0,0 +1,97 @@
+##############################################################################
+#
+# Copyright (c) 2024 Zope Foundation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+"""Setup for zope.meta package
+"""
+
+import os
+
+from setuptools import find_packages
+from setuptools import setup
+
+
+def read(*rnames):
+    with open(os.path.join(os.path.dirname(__file__), *rnames)) as stream:
+        return stream.read()
+
+
+setup(
+    name='zope.meta',
+    version='1.0.dev0',
+    author='Zope Foundation and Contributors',
+    author_email='zope-dev@zope.dev',
+    description='Helper functions for package management',
+    long_description=(
+        read('README.rst')
+        + '\n\n' +
+        read('CHANGES.rst')
+    ),
+    long_description_content_type='text/x-rst',
+    keywords="zope packaging",
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Zope Public License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
```

Comment:
This makes my heart bleed a bit: Previously the minimum requiered version for meta/config was Python 3.11. And I think that I was using Python 3.11+ syntax somewhere. Being forced to write Python 3.8 code in the once most advanced repo feels kind of sad.
But I see the point that it is easier for self updating the repository to support all the versions we currently support for ZF projects.

Reply:
You were using `tomllib`, which (needlessly) introduced the hard Python 3.11 requirement. I replaced that with `tomlkit`, which is used everywhere else in the package. I find it more valuable to have a consistent set of supported versions everywhere. Less surprises.

Reply:
And don't forget, we could not even run the configure script against this package without that consistent Python version support because there's no functionality for a different set.